### ISSUE TITLE
Pagination Items per Page cannot be operated via keyboard

### ISSUE THREAD

**Issue by rlucke** [opencast/opencast-admin-interface#1215]
The element can be accessed via a tab, but no selection can be made. Neither enter and space nor cursor keys work.

**Comment by rlucke** [comment]
I tried the tab button on the element again, you can actually go through the options with it. However, this is no better than if it had not been implemented. Control with the arrow keys would be in line with expectations.

**Comment by Arnei** [comment]
Could you elaborate on this in more detail? I am not sure what exactly you'd like to see here.
